package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "skript",
+  "version": "1.0.0",
```

Comment:
What version would this follow Skript or the website? for the website currently the version is handled [here](https://github.com/SkriptLang/Skript/pull/4319/files#diff-26b330d4ed424c74aa1b75d4ca8936dac30b9c4223f445d39f757c3ec3414453R1)
_In case it doesn't show up from the link_
![image](https://user-images.githubusercontent.com/20037329/201278488-b4f38ebf-c864-486c-a8d9-290857cc9f28.png)